cogctl/cli/upgrade_configuration.py

```diff
@@ -0,0 +1,22 @@
+import click
+
+
+@click.command(name="upgrade-configuration")
+@click.pass_obj
+def upgrade_configuration(state):
```

Comment:
Is there any reason we wouldn't just detect the config file and do the upgrade automatically?